pythonforandroid/bootstraps/common/build/src/main/java/org/kivy/android/PythonService.java

```diff
@@ -16,8 +16,6 @@
 
 import org.kivy.android.PythonUtil;
 
-import org.renpy.android.Hardware;
```

Comment:
Wasn't actually used in this file.
We may introduce some linting to the Java files as well once we're done deleting duplicated code